packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visits-table/visits-table.component.tsx

```diff
@@ -304,7 +305,11 @@ const VisitTable: React.FC<VisitTableProps> = ({ showAllEncounters, visits, pati
                                       className={styles.menuItem}
                                       itemText={t('deleteThisEncounter', 'Delete this encounter')}
                                       onClick={() =>
-                                        handleDeleteEncounter(selectedVisit.id, selectedVisit.form?.display)
+                                        handleDeleteEncounter(
+                                          selectedVisit.visitUuid,
+                                          selectedVisit.id,
```

Comment:
So this suggests that the `selectedVisit.id` is pointing to the encounter UUID, if so, it seems more accurate to rename the variable to something like: `selectedEncounter`.